lib/src/common/managed_stream_subscription.dart

```diff
@@ -19,7 +19,7 @@ class ManagedStreamSubscription<T> implements StreamSubscription<T> {
   Completer<Null> _didComplete = new Completer();
 
   ManagedStreamSubscription(Stream<T> stream, void onData(T),
-      {void onError(error, [stackTrace]), void onDone(), bool cancelOnError})
+      {Function onError, void onDone(), bool cancelOnError})
```

Comment:
- relaxing the type of a method param is NOT breaking
- relaxing the RETURN type of a method IS breaking